tools/python/run_adb.py

```diff
@@ -13,7 +13,22 @@
 
 def run_adb(android_sdk_root: str, args: typing.List[str]):
     sdk_tool_paths = get_sdk_tool_paths(android_sdk_root)
-    run(sdk_tool_paths.adb, *args)
+    if is_emulator_running(sdk_tool_paths.adb):
+        run(sdk_tool_paths.adb, *args)
+    else:
+        print("No emulator is running.")
```

Comment:
I think this should be an error. we don't want it appear to be successful and not actually do anything.

Reply:
Removed this change, let ADO yaml to determine if emulator is running